contrib/sbtc/token-key-admin/contracts/sbtc-token-key-admin.clar

```diff
@@ -0,0 +1,126 @@
+
+;; title: sbtc-token-key-admin
+;; version:
+;; summary:
+;; description:
+
+;; traits
+;;
+(impl-trait 'SP3FBR2AGK5H9QBDH3EEN6DF8EK8JY7RX8QJ5SVTE.sip-010-trait-ft-standard.sip-010-trait)
+
+;; token definitions
+;; 
+(define-fungible-token sbtc u2100000000000000)
+
+;; constants
+;;
+(define-constant contract-owner tx-sender)
+(define-constant err-invalid-caller u1)
+(define-constant err-owner-only u100)
+(define-constant err-not-token-owner u101)
+
+;; data vars
+;;
+(define-data-var coordinator (optional {addr: principal, key: (buff 33)}) none)
+
+;; data maps
+;;
+(define-map signers uint {addr: principal, key: (buff 33)})
+
+;; public functions
+;;
+(define-public (set-coordinator-data (data {addr: principal, key: (buff 33)}))
+    (if (is-valid-caller)
```

Comment:
You can use `asserts!` instead of and if-then-else clause.
https://book.clarity-lang.org/ch06-01-asserts.html

Reply:
We need to wrap with (begin...) to use asserts! properly:

`
    (begin
        (asserts! (is-valid-caller) (err err-invalid-caller))
        (ok (var-set coordinator (some data)))
    )
`
So not saving any space, but happy to do it this way if it's cleaner

Reply:
I think it reads cleaner.

Reply:
Done.